lib/hex_web/web/router.ex

```diff
@@ -100,14 +100,15 @@ defmodule HexWeb.Web.Router do
     conn              = fetch_params(conn)
     packages_per_page = @packages
     search            = conn.params["search"] |> safe_query
+    sort              = conn.params["sort"] |> safe_sort
```

Comment:
Ah of course. Sorry I was in too much of a hurry this afternoon. Thanks for your patience.